crates/primitives/src/transaction/mod.rs

```diff
@@ -836,10 +837,10 @@ impl TransactionSigned {
     /// Returns the enveloped encoded transactions.
     ///
     /// See also [TransactionSigned::encode_enveloped]
-    pub fn envelope_encoded(&self) -> bytes::Bytes {
+    pub fn envelope_encoded(&self) -> Bytes {
```

Comment:
which bytes type is this here?

I'd like to keep bytes::Bytes and not a hex wrapper

Reply:
This is alloy Bytes, I don't see why have multiple types in a public interface